src/ServerRequestFactory.php

```diff
@@ -469,6 +469,7 @@ private static function createUploadedFileFromSpec(array $value)
      */
     private static function normalizeNestedFileSpec(array $files = [])
     {
+        $normalizedFiles = [];
```

Comment:
I stand corrected when running the unit tests. Still trying to understand why this change results in any change in behavior…